javascript/ql/src/Declarations/MixedStaticInstanceThisAccess.ql

```diff
@@ -10,15 +10,18 @@
 */
 import javascript
 
+MethodDefinition getASuperClassMethod(ClassDefinition base, string name) {
```

Comment:
The "super class" bit in the name is misleading. Taking a cue from Java, we could rephrase this as

```suggestion
/** Holds if `base` declares or inherits method `m` with the given `name`. */
predicate hasMethod(ClassDefinition base, string name, Method m) {
```